packages/translations/scripts/translateNewKeys/translateText.ts

```diff
@@ -6,38 +6,68 @@ const dirname = path.dirname(filename)
 
 dotenv.config({ path: path.resolve(dirname, '../../', '.env') })
 
-export async function translateText(text: string, targetLang: string) {
-  const response = await fetch('https://api.openai.com/v1/chat/completions', {
-    body: JSON.stringify({
-      max_tokens: 150,
-      messages: [
-        {
-          content: `Only respond with the translation of the text you receive. The original language is English and the translation language is ${targetLang}. Use formal and professional language. Only respond with the translation - do not say anything else. If you cannot translate the text, respond with "[SKIPPED]"`,
-          role: 'system',
-        },
-        {
-          content: text,
-          role: 'user',
-        },
-      ],
-      model: 'gpt-4',
-    }),
-    headers: {
-      Authorization: `Bearer ${process.env.OPENAI_KEY}`,
-      'Content-Type': 'application/json',
-    },
-    method: 'POST',
-  })
-  try {
-    const data = await response.json()
-    if (data?.choices?.[0]) {
-      console.log('  Old text:', text, 'New text:', data.choices[0].message.content.trim())
-      return data.choices[0].message.content.trim()
-    } else {
-      console.log(`Could not translate: ${text} in lang: ${targetLang}`)
+const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
```

Comment:
This is unrelated to the rest of the PR. We usually try and make smaller PRs for such changes.